hash_practice/exercises.py

```diff
@@ -2,18 +2,50 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(nklog(k)), where k is the length of the string 

```

Comment:
This is a correct answer, but we can make a simplifying assumption here. Since we know the elements of `strings` are English words, their size is rather constrained. On average words in English have about 5 letters on average, which is going to be quickly dwarfed by the number of strings in `strings`, which is unbounded and could be in the hundreds or thousands.

With that simplifying assumption, we can ignore the `k` part of this and just say it is O(n).